README.md

```diff
@@ -133,6 +136,9 @@ The following APIs are shared by Slider and Range.
 | defaultValue | `number[]` | `[0, 0]` | Set initial positions of handles. |
 | value | `number[]` | | Set current positions of handles. |
 | tabIndex | number[] | `[0, 0]` | Set the tabIndex of each handle. |
+| aria-label | Array[string] | - | Set the `aria-label` attribute on each handle. |
```

Comment:
Should the naming perhaps reflect that this is a group of arrays, rather than the HTML attribute `aria-label`?

You could name these something like `ariaLabelGroupForHandles`, `ariaLabelledByGroupForHandles`, and `ariaValueTextFormattersForHandles`